ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/processes/panel/ProcessesPanelPresenter.java

```diff
@@ -674,12 +664,7 @@ public void onStopCommandProcess(ProcessTreeNode node) {
 
     @Override
     public void onCloseCommandOutputClick(final ProcessTreeNode node) {
-        closeCommandOutput(node, new SubPanel.RemoveCallback() {
-            @Override
-            public void remove() {
-                view.hideProcessOutput(node.getId());
-            }
-        });
+        closeCommandOutput(node, () -> {});
```

Comment:
Maybe it would be better to pass `null` as third argument instead of empty callback?